### ISSUE TITLE
formatting on profile updates

### ISSUE THREAD

**Issue by saddington** [yenio/changelog#71]
https://beta.yen.io/nick/status/154160648773972

a list of values is transposed when updating to the profile. 

not sure if this is a bug per se if we're limiting the profile box to be a certain amount of characters. if so, we should limit the description update box accordingly to match the format

**Comment by saddington** [comment]
@james-yen does nick want those paragraph breaks to stay?

**Comment by james-yen** [comment]
yes, in the change box the values are vertical but when updating they go horizontal on the actual profile. 

people often times list out interests in lists on their profile and right now it saves them with no line/paragraph breaks

**Comment by saddington** [comment]
kk. that's... interesting. sure, why the hell not.
